lisa/sut_orchestrator/azure/core_stats.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
The file name should be called `notifiers`. All Azure specified notifiers can be in it.

Reply:
Done